app/Models/RH/Equipement.php

```
@@ -66,6 +67,11 @@ public function maintenanceTickets(): MorphMany
         return $this->morphMany(AssetMaintenanceTicket::class, 'asset');
     }
 
+    public function trackings(): HasMany
+    {
+        return $this->morphMany(ChantierEquipmentTracking::class, 'trackable');
```

PHPStan (2.2.8)

[error] Method App\Models\RH\Equipement::trackings() should return Illuminate\Database\Eloquent\Relations\HasMany but returns Illuminate\Database\Eloquent\Relations\MorphMany<App\Models\Chantiers\ChantierEquipmentTracking, $this(App\Models\RH\Equipement)>.


(return.type)